bin/spring

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
```

Comment:
does your gem file have spring in it? when you create new project.  I thought the purpose of the spring is to keep the application alive in the background whilst changes are being made.  I don't remember explicitly adding this.  It may have come in when i did bundle install or rspec install